Makefile

```diff
@@ -65,13 +74,13 @@ ifeq ($(OPENEDX_ATLAS_PULL),)
 	git clean -fdX conf/locale/rtl
 	git clean -fdX conf/locale/eo
 	i18n_tool validate --verbose
-	paver i18n_compilejs
 else
+	make pull_xblock_translations
 	find conf/locale -mindepth 1 -maxdepth 1 -type d -exec rm -r {} \;
-	atlas pull $(OPENEDX_ATLAS_ARGS) translations/edx-platform/conf/locale:conf/locale
+	atlas pull $(ATLAS_OPTIONS) translations/edx-platform/conf/locale:conf/locale
```

Comment:
Thanks @feanil for the quick triage.

Conflicts has been resolved and I've tested it again and it seems to be playing well. 

I've changed this line to match the Tutor option to have one less surprise for operators.